### PR TITLE
Dtsw 5312 investigate format discrepancy of extrinsic calibration for backported daffy

### PR DESCRIPTION
**Pull request by Tuxliri**

### CODE REVIEW

File: duckiebot/calibrate_extrinsics/command.py [hunk 1]
```diff
@@ -1,91 +1,115 @@
...
+BRANCH = "daffy"
+DEFAULT_IMAGE = "duckietown/dt-gui-tools:" + BRANCH + "-" + ARCH
 
 
 class DTCommand(DTCommandAbs):
     @staticmethod
     def command(shell: DTShell, args):
 
-        prog = "dts duckiebot calibrate_extrinsics DUCKIEBOT_NAME"
+        prog = "dts duckiebot calibrate_extrisics DUCKIEBOT_NAME"
```
Comment (afdaniele) [inline duckiebot/calibrate_extrinsics/command.py]: typo

File: duckiebot/calibrate_extrinsics/command.py [hunk 2]
```diff
@@ -1,91 +1,115 @@
...
         usage = """
 Calibrate:
 
     %(prog)s
 """
 
         parser = argparse.ArgumentParser(prog=prog, usage=usage)
-        parser.add_argument("duckiebot", default=None, help="Name of the Duckiebot to calibrate")
+        parser.add_argument("hostname", default=None, help="Name of the Duckiebot to calibrate")
```
Comment (afdaniele) [inline duckiebot/calibrate_extrinsics/command.py]: should be `duckiebot`

File: duckiebot/calibrate_extrinsics/command.py [hunk 3]
```diff
@@ -1,91 +1,115 @@
...
-            privileged=True,
-            network_mode="host",
-            volumes=bind_duckiebot_data_dir(),
-            command="dt-launcher-calibrate-extrinsics",
-            environment=env,
-            remove=True,
+        dtslogger.info(
+            "When the window opens you will be able to perform the calibration.\n "
+            "Follow the instructions on the official book at https://docs.duckietown.com/daffy/"
+            "opmanual-duckiebot/operations/calibration_camera/index.html#extrinsic-camera-calibration.\n "
```
Comment (afdaniele) [inline duckiebot/calibrate_extrinsics/command.py:95]: let's make sure the docs are updated

**Review by afdaniele — CHANGES_REQUESTED** [review]

File: duckiebot/calibrate_extrinsics/command.py (hunk 3 above)
Comment (Tuxliri) [reply duckiebot/calibrate_extrinsics/command.py:95]: This has been completed.

**Pull request merged**